AISKU/package.json

```diff
@@ -1,9 +1,10 @@
 {
     "name": "@microsoft/applicationinsights-web",
-    "version": "1.0.0-beta.2",
+    "version": "1.0.0-beta",
     "description": "Microsoft Application Insights Javascript SDK API 1.0 beta",
-    "main": "dist-esm/applicationinsights-sdk-js.js",
-    "types": "types/applicationinsights-sdk-js.d.ts",
+    "main": "dist/applicationinsights-web.min.js",
+    "module": "dist-esm/applicationinsights-web.js",
+    "types": "types/applicationinsights-web.d.ts",
```

Comment:
totally unrelated to pr: did the api extractor plugin work to roll up interfaces to single file?

Reply:
It left all of the external types as external instead of trying to resolve them